.github/workflows/allthekeeps-testnet.yml

```diff
@@ -0,0 +1,103 @@
+name: AllTheKeeps / Testnet
```

Comment:
Can we have one workflow for any network `.github/workflows/allthekeeps.yml`?
```suggestion
name: AllTheKeeps
```

Reply:
This particular workflow handles specifically Ropsten network. All the scripts that are triggered and dApp is pointing to Ropsten. ATK on mainnet is in hands of community.